src/main/java/org/broadinstitute/gatk/nativebindings/NativeLibrary.java

```diff
@@ -0,0 +1,21 @@
+package org.broadinstitute.gatk.nativebindings;
```

Comment:
Should this just be org.broadinstitute.nativebinding ?  It's not really gatk specific.